### ISSUE TITLE
tailwind-elements - adding custom css

### ISSUE THREAD

**Issue by robokonk** [mdbootstrap/TW-Elements#1236]
I am using this carousel:  
[https://tailwind-elements.com/docs/standard/components/carousel/](https://tailwind-elements.com/docs/standard/components/carousel/)

I don't know how to add custom CSS, instead default for bellow div:  
```
<div id="carouselExampleCaptions" class="carousel slide relative" data-bs-ride="carousel">

 <div class="carousel-indicators absolute right-0 bottom-0 left-0 flex justify-center p-0 mb-4">
 ...
```

I should copy `carousel-indicators` to my CSS, and change it?
How practice is the best?

**Comment by smolenski-mikolaj** [comment]
We've just released the new v1.0.0-beta1 version where we use almost only tailwind class system and all elements can be customize. Please update and check it out! 

https://tailwind-elements.com/docs/standard/components/carousel/

**Comment by smolenski-mikolaj** [comment]
Closing due to inactivity.
